dotnet/src/SemanticKernel.Abstractions/Orchestration/FunctionResult.cs

```diff
@@ -47,31 +51,31 @@ public sealed class FunctionResult
     internal object? Value { get; private set; } = null;
 
     /// <summary>
-    /// Instance of <see cref="ContextVariables"/> to pass in function pipeline.
+    /// The culture configured on the Kernel that executed the function.
     /// </summary>
-    internal ContextVariables Variables { get; private set; }
+    internal CultureInfo Culture { get; set; }
 
     /// <summary>
     /// Initializes a new instance of the <see cref="FunctionResult"/> class.
     /// </summary>
     /// <param name="functionName">Name of executed function.</param>
-    /// <param name="variables">Instance of <see cref="ContextVariables"/> to pass in function pipeline.</param>
-    public FunctionResult(string functionName, ContextVariables variables)
+    public FunctionResult(string functionName)
     {
         this.FunctionName = functionName;
-        this.Variables = variables;
+        this.Culture = CultureInfo.InvariantCulture;
     }
 
     /// <summary>
     /// Initializes a new instance of the <see cref="FunctionResult"/> class.
     /// </summary>
     /// <param name="functionName">Name of executed function.</param>
-    /// <param name="variables">Instance of <see cref="ContextVariables"/> to pass in function pipeline.</param>
     /// <param name="value">Function result object.</param>
-    public FunctionResult(string functionName, ContextVariables variables, object? value)
-        : this(functionName, variables)
+    /// <param name="culture">The culture configured on the Kernel that executed the function.</param>
+    public FunctionResult(string functionName, object? value, CultureInfo culture)
```

Comment:
Rather than pulling out the culture specifically, how about just passing in and storing the Kernel?